src/default_out.rs

```diff
@@ -1140,9 +1141,15 @@ pub fn dump_undefined_struct(
                     number_of_installable_languages
                 );
             }
+            let mut langs_installed = 0;
             for installable_language in data.installable_langauges() {
+                langs_installed += 1;
                 println!("\t\t{}", installable_language);
             }
+            // TODO: done to preserve behavior of dmidecode
+            for _ in langs_installed..data.number_of_installable_languages().unwrap_or(u8::MAX) {
+                println!("\t\t{}", "<BAD INDEX>");
```

Comment:
We're still missing a complete solution for <BAD INDEX>, I'm thinking anything that returns an Option<String>, we unwrap_or("<BAD INDEX>"). Though I'll have to cross check if dmidecode does it that way every time.